src/hermes_claude_agent_sdk/parity/v4_local_mechanism_executor.py

```
@@ -0,0 +1,467 @@
+        "kind": kind,
+        "byte_length": len(encoded),
+        "sha256": _digest(encoded),
+        "terminal_status": terminal,
+    }
+
+
+def _admit(row_key: str, trial_index: int, path: str) -> tuple[Any, Mapping[str, Any]]:
+    if not isinstance(row_key, str) or not isinstance(path, str) or path not in _PATHS or type(trial_index) is not int:
+        _fail("row, trial, or path is unsupported")
+    try:
+        document, contract, materializer, catalog = _frozen_inputs()
```

Ruff (0.16.3)

[warning] Unpacked variable `document` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)